pdc/apps/common/renderers.py

```diff
@@ -65,6 +65,8 @@
 """
 
 URL_SPEC_RE = re.compile(r'\$(?P<type>URL|LINK):(?P<details>[^$]+)\$')
+ORDERING_STRING = "\n \n * __Ordering Field__ is used to override the ordering of the results by client, " \
```

Comment:
is it ok to change Ordering Field to ordering and make it the same style/format with other query params